doc/rados/configuration/auth-config-ref.rst

```diff
@@ -205,34 +176,11 @@ To perform this step manually, execute the following::
 You may specify the key itself in the Ceph configuration file using the ``key``
 setting (not recommended), or a path to a keyfile using the ``keyfile`` setting.
 
-
 .. confval:: keyring
-
-   The path to the keyring file.
-
-  :type: String
-  :required: No
-  :default: ``/etc/ceph/$cluster.$name.keyring,/etc/ceph/$cluster.keyring,/etc/ceph/keyring,/etc/ceph/keyring.bin``
-
-
+   :default: /etc/ceph/$cluster.$name.keyring,/etc/ceph/$cluster.keyring,/etc/ceph/keyring,/etc/ceph/keyring.bin
```

Comment:
this is probably worth a comment in the yaml file, so if we change the default there we know to update the docs. unless you're planning to make this default automatic in the future as well

Reply:
@jdurin thank you for your review. documented. it's difficult to access cmake variables when building document, and i want to keep the subvar mechinary in cmake. so i have no plan to implement this at this moment.

Reply:
thanks, that makes sense wrt to cmake